setup.sh

```diff
@@ -49,11 +49,11 @@ install(){
 # Run tasks to build the project for distribution.
 build(){
   echo 'Building project...'
-  grunt clean
-  grunt
+  gulp clean
```

Comment:
Ooo, sorry I misunderstood. I thought you were saying recreate the clean task here instead of inside gulp. So the reason `gulp clean` is in `setup.sh`'s build step is that `setup.sh`'s steps are around handling project dependencies, whereas the gulp steps are a level higher and are handling build processes after all project dependencies are present. In theory a dependency for gulp could be installed via npm (or gulp itself could be installed via npm), in which case if `gulp clean` was in `setup.sh`'s clean step it would run before the npm dependencies were installed via the install step.